src/containers/Instructor.js

```diff
@@ -181,6 +182,14 @@ function Instructor (props) {
                   </Paper>
                 ))} />
             </> : <Spinner />}
+            <VictoryChart domainPadding={20}>
+              <VictoryBoxPlot boxWidth={20} data={[
```

Comment:
Will this data be replaced with json later?

Reply:
Ya, it'll be replaced with data fed via API in the future. This data's the sample data that Victory had in their docs.